contents/blog/open-source-telemetry-ethical.md

```diff
@@ -0,0 +1,102 @@
+---
+date: 2020-09-21
+title: Should that open source project track you?
```

Comment:
I get what you're going for here, but I think it sounds a bit confusing to the reader. Maybe just "Should open source projects track you?"